pandas/tests/plotting/test_datetimelike.py

```diff
@@ -49,7 +51,15 @@ def test_ts_plot_with_tz(self, tz_aware_fixture):
         tz = tz_aware_fixture
         index = date_range("1/1/2011", periods=2, freq="H", tz=tz)
         ts = Series([188.5, 328.25], index=index)
-        _check_plot_works(ts.plot)
+        if tz in ["UTC", pytz.UTC, timezone.utc] or isinstance(
+            tz, dateutil.tz.tz.tzutc
```

Comment:
1) this should use tz_compare or is_utc (though I think there's an issue about the string not being handled correctly)

2) more important, shouldn't we expect to get this warning for _any_ tzaware, not just UTC?

Reply:
> more important, shouldn't we expect to get this warning for any tzaware, not just UTC?

Apparently we consider it a non-ts plot when the tz is non-UTC. See `_use_dynamic_x`.

Debugging that is probably out of scope for this PR. I think just filtering the warning regardless of the tz is probably OK.

Reply:
Sure, I could add `@pytest.mark.filterwarnings`. Everyone okay with that?

Reply:
I think that would be fine